apps/wallet-mobile/src/features/Menu/Menu.tsx

```diff
@@ -198,6 +205,7 @@ const useNavigateTo = () => {
         },
       })
     },
+    staking: () => navigateToStakingDashboard(),
```

Comment:
if dashboard is a screen it should be called DashboardScreen and be wrapped in a SafeAreView (when it is not happening in the navigator)

Reply:
I just copied what the footer link had and it worked. 
I'll rename it and look into the SafeAreaView.

Reply:
This was changed in another branch and now applied from develop